yarn.lock

```diff
@@ -3170,11 +3195,6 @@ boolbase@^1.0.0, boolbase@~1.0.0:
   resolved "https://registry.yarnpkg.com/boolbase/-/boolbase-1.0.0.tgz#68dff5fbe60c51eb37725ea9e3ed310dcc1e776e"
   integrity sha1-aN/1++YMUes3cl6p4+0xDcwed24=
 
-bootstrap@^4.4.1:
```

Comment:
Is there a reason bootstrap was removed. I think it is a dependency for the fhir visualizers

Reply:
only thing that changed in the `package.json` was the addition of semantic-ui.  can't speak for why that changed...

any ideas @dehall?  Maybe it is related to getting `fhir-visualizers` through npm now (I'd imagine it should bring whatever dependencies it needs without us having to specify)

Reply:
No clue. fhir-visualizers has been a pain to get working correctly. I'll try a fresh clone of this branch and see if it loads correctly.

Reply:
So no it's not using bootstrap as expected -- the patient record UI would be more consistently spaced and have row-level highlighting on hover if it did -- though it's also not failing to load the app completely like we've seen before when bootstrap wasn't included. I suspect it's an issue with the fhir-visualizers npm package, not with us using it here. Also we're going to get a new design for the patient record on seen anyway, so we can investigate further when we work on that new design.